CHANGELOG.md

```diff
@@ -1,14 +1,35 @@
 # Change Log for rocm-cmake
 
+## [0.6.0]
+### Added
+- `rocm_set_devel_component` can be used to set the component used for devel packaging (defaults to devel). 
```

Comment:
I think it would be better to have an opt-out instead of an opt-in setting. I think it would be better to have a global cached variable such as `ROCM_ENABLE_DEV_COMPONENTS`, and libraries that wont work with this can either set it to off before calling `find_package(ROCM)` or when calling cmake(ie `cmake -DROCM_ENABLE_DEV_COMPONENTS=Off`).

Reply:
I agree that is a nicer solution, the justification behind having an opt-in system is to maintain backwards compatibility. Opt-out would be a breaking change in behaviour, so anyone building for older versions of ROCm would have issues. That said, I'm not sure how much of a concern backwards compatibility is.

Reply:
With a cached variable, the user can just pass `-DROCM_ENABLE_DEV_COMPONENTS=Off` to build an older library with a newer version of rocm-cmake.